common/rpc/test/rpc_common_test.go

```diff
@@ -100,6 +100,7 @@ func startHelloWorldServer(s *suite.Suite, factory *TestFactory) (*grpc.Server,
 	helloworld.RegisterGreeterServer(server, greeter)
 
 	listener := factory.GetGRPCListener()
+	s.NotEqual(rpcTestCfgDefault.GRPCPort, 0)
```

Comment:
To test this properly, you'll need to add a test which sets the port to 0, starts the server, and then verifies that it is set to a non-zero number

Reply:
This is using 0 (see https://github.com/temporalio/temporal/pull/4411/files/9012b7924b841a6ea351e734ada3296aff35d40a#diff-8f16f53e01a0670b51e43596ee903cd2dd83e51399714ff96c0e5da69e4d3448R76)

but I understand we might want to separate the unit test for the purpose. Where should this unit test reside?